### PR TITLE
[MWPW-159905] - Add config for full width and no border along with jarvis fix for standalone GNAV

### PR DESCRIPTION
**Pull request by Deva309**
Things covered as part of this PR:
1) Add config as `layout` for supporting full width in Standalone GNAV.
2) Add config for `noBorder` for removing border for adobe home.
3) Relative url href removal added for custom Links
4) Add Jarvis call for standalone GNAV

Resolves: 
[MWPW-159905](https://jira.corp.adobe.com/browse/MWPW-159905)
[MWPW-160508](https://jira.corp.adobe.com/browse/MWPW-160508)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mwpw-159905--milo--deva309.hlx.page/?martech=off

QA:

- https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-159905&authoringpath=/federal/test-website&layout=fullWidth (full width)
- https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-159905&authoringpath=/federal/test-website&layout=fullWidth&noborder=true (no border bottom)

### CODE REVIEW

File: libs/navigation/navigation.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const blockConfig = [
     name: 'global-navigation',
     targetEl: 'header',
     appendType: 'prepend',
-    params: ['imsClientId', 'searchEnabled', 'unav', 'customLinks', 'jarvis'],
+    params: ['imsClientId', 'searchEnabled', 'unav', 'customLinks', 'jarvis', 'layout'],
```
Comment (bandana147) [inline libs/navigation/navigation.js:7]: You don't need to pass the param layout since it is not used in the block code through milo configs.

Comment (Deva309) [reply libs/navigation/navigation.js]: Done removed from params.

File: libs/navigation/bootstrapper.js [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+  const { name, targetEl, layout, noBorder } = blockConfig;
   const { getConfig, createTag, loadLink, loadScript } = await import(`${miloLibs}/utils/utils.js`);
   const { default: initBlock } = await import(`${miloLibs}/blocks/${name}/${name}.js`);
 
   const styles = [`${miloLibs}/blocks/${name}/${name}.css`, `${miloLibs}/navigation/navigation.css`];
   styles.forEach((url) => loadLink(url, { rel: 'stylesheet' }));
 
+  const setNavLayout = () => {
+    const element = document.querySelector(targetEl);
+    if (element && layout === 'fullWidth') {
```
Comment (bandana147) [inline libs/navigation/bootstrapper.js:11]: The element will always exist, so I believe we can skip the element check here.

**Review by bandana147 — APPROVED** [review]

**Comment by spadmasa** [comment]
Validation done on the pr :
https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-159905&authoringpath=/federal/test-website&layout=fullWidth
{C9C76679-B3B3-42DA-9FBD-0DFD28EA2255}
https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-159905&authoringpath=/federal/test-website&layout=fullWidth&noborder=true
{17E5CC74-6B23-4B4E-81C9-D468C31491E7}
https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-159905&authoringpath=/federal/dev/soujanya&layout=fullWidth&noborder=true&customlinks=home,apps,learn,files,discover
{4FBFFFD9-58CB-4D17-9FB8-8EC889D3C08A}
{144618A7-9DF0-43F7-AE4A-32041CA6A967}

**Pull request merged**